server/src/main/java/com/google/coffeehouse/util/IdentifierGenerator.java

```diff
@@ -12,19 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.coffeehouse;
+package com.google.coffeehouse.util;
 
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
+/**
+ * This interface will be implemented by objects who can generate string IDs.
+ */
+public interface IdentifierGenerator {
```

Comment:
Was there a reason this was branched from a test?

Reply:
No, That was just my mistake :)